ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -343,16 +350,21 @@ import { DraggableListBehavior } from '../nuxeo-draggable-list-behavior.js';
                       ></nuxeo-data-table-row-detail>
                     </template>
                   </dom-if>
-                  <nuxeo-data-table-row-actions
-                    index="[[index]]"
-                    editable="[[editable]]"
-                    orderable="[[orderable]]"
-                    template="[[rowForm]]"
-                    item="[[item]]"
-                    size="[[items.length]]"
-                    table="[[_this]]"
-                  >
-                  </nuxeo-data-table-row-actions>
+                  <dom-if if="[[_hasActions()]]">
+                    <template>
+                      <nuxeo-data-table-row-actions
+                        index="[[index]]"
+                        editable="[[editable]]"
+                        orderable="[[orderable]]"
+                        template="[[rowForm]]"
+                        item="[[item]]"
+                        size="[[items.length]]"
+                        table="[[_this]]"
+                        actions$="[[_setActionsColumnSize(editable, orderable)]]"
```

Comment:
Since the properties used to compute actions are forwarded to `nuxeo-data-table-row-actions`, could we consider to have that logic inside?